bemani/backend/gf/gfv8.py

```diff
@@ -0,0 +1,2657 @@
+import binascii
+import copy
+import base64
+from collections import Iterable
+from typing import Optional, Any, Dict, List, Sequence, Union, Tuple
+
+from bemani.backend.gf.base import GuitarFreaksBase
+from bemani.backend.ess import EventLogHandler
+from bemani.common import ValidatedDict, GameConstants, VersionConstants, Time, ID, intish
+from bemani.data import UserID, Score, Data
+from bemani.protocol import Node
+
+from bemani.backend.gf.gfv7 import GuitarFreaksV7
+
+class GuitarFreaksV8(
+    EventLogHandler,
+    GuitarFreaksBase,
+):
+
+    name = "GuitarFreaks V8"
+    version = VersionConstants.GUITARFREAKS_V8
+
+    GAME_LIMITED_LOCKED = 1
+    GAME_LIMITED_UNLOCKED = 2
+
+    GAME_CURRENCY_PACKETS = 0
```

Comment:
Looks like SDVX copypasta here too? I skipped reviewing gfv4-v7 because I assume you did the same thing with them as you did with DM.